crates/gpui/src/platform/linux/wayland/client.rs

```diff
@@ -403,6 +423,10 @@ impl Dispatch<wl_keyboard::WlKeyboard, ()> for WaylandClientState {
                             } else if key_sym == xkb::Keysym::Alt_L || key_sym == xkb::Keysym::Alt_R
                             {
                                 state.modifiers.alt = true;
+                            } else if key_sym == xkb::Keysym::Super_L
+                                || key_sym == xkb::Keysym::Super_R
+                            {
+                                state.modifiers.command = true;
```

Comment:
For the time being, this allows to test keyboard shortcuts (and features) by using `super` instead of `cmd` as there isn't a linux keymap yet.

Not sure what behavior we want, but we could use the same convention as the web: both the super and command keys map to the "meta" modifier.

Reply:
It doesn't seem like a good desicion because it doesn't work on all platforms (e.g. Gnome)